index.js

```diff
@@ -0,0 +1,46 @@
+const navToggle = document.querySelector('#navToggle');
+const navIcon = document.querySelectorAll('.navIcon');
+const nav = document.querySelector('nav');
+
+navToggle.addEventListener('click', () => {
+  nav.classList.toggle('open');
+  navIcon.forEach((icon) => {
+    icon.classList.toggle('hidden');
+  });
+});
+//  formValidation
+const email = document.getElementById('email');
+const form = document.getElementById('input');
+const displayMsg = document.getElementById('form-message-error');
+form.addEventListener('submit', (event) => {
+  if (email.value !== email.value.toLowerCase()) {
```

Comment:
- [ ] [Optional] By convention email are always in lowercase, it would be a better user experience to internally convert to lower case to given input